source/wp-content/themes/wporg-parent-2021/inc/block-styles.php

```diff
@@ -117,7 +117,7 @@ function setup_block_styles() {
 		'core/heading',
 		array(
 			'name'         => 'with-arrow',
-			'label'        => __( 'With Arrow', 'wporg' ),
+			'label'        => __( 'CTA', 'wporg' ),
```

Comment:
This is up for debate but I think it makes it a little more obvious that this heading style is expected to have an link applied

Reply:
I'd prefer to keep it descriptive of the pattern, since we might have other styles that are "calls to action". How about "Link with arrow" instead? Then the class name is still consistent.

Reply:
I did try that but the text is too long and becomes truncated

![Screen Shot 2022-08-23 at 4 40 15 PM](https://user-images.githubusercontent.com/1017872/186071166-7e74132a-4aac-486b-80c7-b6d262b6e146.jpg)

'Link arrow' fits

![Screen Shot 2022-08-23 at 4 42 32 PM](https://user-images.githubusercontent.com/1017872/186071482-ca4be0b3-d51c-41c4-978f-8ff458e91475.jpg)

Or I guess we could add styles to wrap the text?

Reply:
No, if we override the styles here it might have other effects/conflicts in the future. I tried Link & Arrow and that isn't cut off 👍🏻